spec/services/merge/merge_organisations_service_spec.rb

```diff
@@ -707,6 +707,28 @@
             expect(owned_lettings_log.managing_organisation).to eq(absorbing_organisation)
           end
 
+          context "with merging organisation with schemes with locations" do
+            let!(:scheme) { create(:scheme, owning_organisation: merging_organisation) }
+            let!(:incomplete_location) { create(:location, scheme:, name: nil, postcode: nil) }
+            let!(:complete_location) { create(:location, scheme:) }
+            let!(:complete_location2) { create(:location, scheme:) }
+
+            context "with lettings log with scheme set but no location" do
+              let!(:incomplete_lettings_log) { build(:lettings_log, scheme:, owning_organisation: merging_organisation, startdate: Time.zone.today) }
+
+              it "does not change the lettings log location" do
+                incomplete_lettings_log.save!(validate: false)
+                expect(Rails.logger).not_to receive(:error)
```

Comment:
Feels like this is testing something different? Doesn't need to be removed I guess but a little strange to be part of this test.

Reply:
oh right I can restructure this as the layout is a bit confusing
- `incomplete_lettings_log.save!(validate: false)` yeah we're not testing this line. just much easier to setup a log like this & not have to set a bunch of unrelated properties to satisfy the validation rules
- the expect keyword has to go before the function call. its not verifying the line above
have reworked things to be a little clearer

Reply:
Yeah much clearer now, thanks!